web/.yarnrc.yml

```diff
@@ -1 +1,2 @@
 nodeLinker: node-modules
+enableHardenedMode: false
```

Comment:
Why this change?

Reply:
temp thing for to make ci run faster. taking out now.